### DISCUSSION TITLE
View Notes Added in Other PDF Programs

### DISCUSSION THREAD

**Discussion by gnuarm** [sumatrapdfreader/sumatrapdf#3063] [Documentation]
I use Foxit Reader to add Notes to PDF files.  I can fully view these notes in Foxit.  In Sumatra, only the Note marker icon is visible and I can't find a way to read the note.  This icon looks like a tiny text balloon in yellow, like a tiny sticky note.  

In Foxit Reader you can hover over it to read the text, or it can be double clicked to open it for reading and editing.  Is the ability to view these notes not part of Sumatra?   

v3.4.14267 64 bit Pre-release 

BTW, what is an "issue"?

**Comment by GitHubRulesOK** [comment]
what is an "issue"?
very good question it should NOT be I would like SumatraPDF to do this like my browser,
however it could be I expected text hover to show the tip in latest version


![image](https://user-images.githubusercontent.com/12754913/198855686-2b3f062a-3e5e-455d-9fe5-a1e10fa3d97c.png)

**Reply by gnuarm** [reply]
Sorry, I don't know what you are referring to about the "browser".  I didn't say anything about viewing PDFs in the browser.  

The apparent lines of reasoning by those involved with Sumatra are a bit interesting.  I remember once asking that Sumatra could implement a similar handling of double click and drag like most programs do.  The reasoning for limiting the selection to a single word was along the lines of "That's how Acrobat does it", which makes no sense to me.  

Here, if I understand what you are saying, you don't think this should be implemented like other PDF viewers, like Foxit?

**Reply by GitHubRulesOK** [reply]
Every app has different abilities otherwise adobe and foxit teams would be in same room (very big room as they both  employ 100s/10000s programmers) SumatraPDF uses MuPDF lightweight engine here on the left
so like hover in Edge (Foxit PDF viewer) in the center
SumatraPDF uses MSVC features to enhance those short comments to enhance by display as tooltips

![image](https://user-images.githubusercontent.com/12754913/198875752-e1211eba-47e8-47ae-a0be-660400a8664d.png)

**Comment by gnuarm** [comment]
We don't seem to be communicating at all.  

Bottom line is, I can't find a useful way to display these notes in Sumatra.  It would be a nice feature to display them in a useful manner.  

That's all I'm saying.  I don't actually expect anything to happen.  It's pretty clear that the people who could make this happen don't think like users.  I've seen this on other suggestions.  Sometimes they insist they need to do what other PDF viewers do and sometimes they say there's no reason to do what other PDF viewers do.

**Comment by tactilis** [comment]
@gnuarm
> I can't find a useful way to display these notes in Sumatra

In the latest version of Sumatra PDF (v3.4.6), hovering over the comment icon shows comment text that I added using Acrobat.
 
![image](https://user-images.githubusercontent.com/4060092/198893775-2cf038d5-dd40-4f6f-8b58-32deb681e5b0.png)
 
![image](https://user-images.githubusercontent.com/4060092/198893785-7a136917-e280-44b0-a782-bfac6646bb8b.png)
 
The version of SumatraPDF you are running (v3.4.14267 64 bit Pre-release) is out of date.

**Comment by GitHubRulesOK** [comment]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/300a7a5b-f1d1-4516-8bec-88ecd026c063)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c6d75780-60f8-422a-9ee9-2424afc7627e)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c6cad35c-e4e4-4f73-bf66-5d981d772528)
